src/components/v5/shared/Avatar/Avatar.tsx

```diff
@@ -44,7 +45,11 @@ const Avatar: FC<PropsWithChildren<AvatarProps>> = ({
 
   return (
     <figure
-      className={className ? `${mainClass} ${className}` : mainClass}
+      className={
+        className || borderClassName
+          ? `${borderClassName} ${mainClass} ${className}`
```

Comment:
You will get undefined if borderClassName or className won't be passed. Also, what is this class needed for? Isn't it ok to just use `className`?